lucet-runtime/lucet-runtime-internals/src/instance.rs

```diff
@@ -76,6 +76,7 @@ pub fn new_instance_handle(
     module: Arc<dyn Module>,
     alloc: Alloc,
     embed_ctx: CtxMap,
+    limiter: Option<Box<dyn ResourceLimiter>>,
```

Comment:
This should probably be an `Arc` instead of a `Box` to make integration with other parts of a system easier and more efficient.